handlers/app.go

```diff
@@ -0,0 +1,48 @@
+package handlers
+
+
+import (
+
+	"github.com/knative-scout/app-api/models"
+	"go.mongodb.org/mongo-driver/bson"
+	"net/http"
+	"github.com/gorilla/mux"
+	"fmt"
+)
+
+// AppByIDHandler returns a single app by ID from the database
+type AppByIDHandler struct {
+	BaseHandler
+}
+// ServeHTTP implements http.Handler
+func (h AppByIDHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+
+	//gets all the optional parametres passed in the URL
+	vars := mux.Vars(r)
+
+	id := vars["id"]
+
+	resp := getDataFromDB(id, h)
+
+	h.RespondJSON(w, http.StatusOK, resp)
```

Comment:
Can you please make the endpoint return data in the format which the spec documented? https://github.com/knative-scout/app-api/blob/master/DESIGN.md#get-app-by-id

It should return an object with an `app` key